src/coreComponents/mesh/mpiCommunications/mpiCommunicationUtils.hpp

```diff
@@ -0,0 +1,133 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file CommID.hpp
+ */
+
+#ifndef GEOSX_MESH_MPICOMMUNICATIONS_MPICOMMUNICATIONUTILS_HPP_
+#define GEOSX_MESH_MPICOMMUNICATIONS_MPICOMMUNICATIONUTILS_HPP_
+
+#include "common/DataTypes.hpp"
+
+namespace geosx
+{
+
+struct SyncFieldsID
```

Comment:
```suggestion
class fieldIdentifiers
```

Is there anything about this that is specific to the communications or syncing of fields?

Reply:
just the way is used t.b.h.